.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+  "name": "Stellar Docs - Docusaurus/Yarn",
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:1-22",
+  "hostRequirements": {
+    "cpus": 2
+  },
+  "forwardPorts": [
+    3000
+  ],
+  "portsAttributes": {
+    "3000": {
+      "elevateIfNeeded": true,
+      "onAutoForward": "openBrowser",
+      "label": "docusaurus",
+      "requireLocalPort": true
+    }
+  },
+  // Configure tool-specific properties.
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "[typescript]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode",
+          "editor.formatOnSave": true
+        },
+        "[json]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode",
+          "editor.formatOnSave": true
+        },
+        "[jsonc]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode",
+          "editor.formatOnSave": true
+        }
+      },
+      "extensions": [
+        "dbaeumer.vscode-eslint",
+        "orta.vscode-jest",
+        "esbenp.prettier-vscode",
+        "streetsidesoftware.code-spell-checker"
+      ]
+    }
+  },
+  "postCreateCommand": "yarn install",
+  waitFor: "postCreateCommand",
+  "updateContentCommand": "npx docusaurus start"
```

Comment:
is there a reason to use `npx ...` rather than `yarn ...`?

Reply:
Oh thats what it says in the README haha yea `yarn` is better